build/perf-test-image/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM gcr.io/cloud-builders/gcloud
```

Comment:
Doesn't the build image already have all these dependencies? Just questioning if we need another image to the set?

Reply:
Yes build image has the necessary dependencies but it also have many dependencies which is not required for this task. These dependencies add around 5 more minutes to the total build time. That's the only downside I see now. I wrote theses steps in this way to aline with what we did for e2e test, and I don't have strong objection to switch to build image though.

Reply:
You don't need to build the image though - you could pull the prebuilt image we build and cache, and run that instead.

Copy this section to do it:

https://github.com/googleforgames/agones/blob/93a4c4ab663d9a19f65959920d0a71408cabe457/cloudbuild.yaml#L76-L83